docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -1,184 +1,262 @@
+If you are already running the Teleport Auth Service and Proxy Service on
+another platform, you can use your existing Teleport deployment to access your
+Kubernetes cluster. [Follow our
 guide](../../kubernetes-access/getting-started.mdx) to connect your Kubernetes
 cluster to Teleport.
 
 (!docs/pages/includes/cloud/call-to-action.mdx!)
 
-## Follow along with our video guide
+## Prerequisites
 
-<iframe
-  width="712"
-  height="400"
-  src="https://www.youtube-nocookie.com/embed/VPGYLEMTdJ8?rel=0&modestbranding=1"
-  frameBorder="0"
-  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-  allowFullScreen
-/>
+- A registered domain name. This is required for Teleport to set up TLS via
+  Let's Encrypt and for Teleport clients to verify the Proxy Service host.
+- A Kubernetes cluster hosted by a cloud provider, which is required for the
+  load balancer we deploy in this guide.
+- A persistent volume that the Auth Service can use for storing cluster state.
+  Make sure your Kubernetes cluster has one available:
 
-## Prerequisites
-- A registered domain name. This is required for Teleport to set up TLS via Let's Encrypt and for Teleport clients to verify the Proxy Service host.
-- A Kubernetes cluster hosted by a cloud provider, which is required for the load balancer we deploy in this guide.
+  ```code
+  $ kubectl get pv
+  ```
+  
+  If there are no persistent volumes available, you will need to either provide
+  one or enable [dynamic volume
+  provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#enabling-dynamic-provisioning)
+  for your cluster. For example, in Amazon Elastic Kubernetes Service, you can
+  configure the [Elastic Block Store Container Storage Interface driver
+  add-on](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html).
+
+  To tell whether you have dynamic volume provisioning enabled, check for the
+  presence of a `StorageClass`:
+
+  ```code
+  $ kubectl get storageclasses
+  ```
+
+  <Details title="Launching a fresh EKS cluster with eksctl?" open="false">
+
+  If you are using [eksctl](https://eksctl.io/) to launch a fresh Amazon Elastic
+  Kubernetes Service cluster in order to follow this guide, the following
+  example configuration sets up the EBS CSI driver add-on. Update the cluster
+  name, version, node group size, and region as required:
+
+  ```yaml
+    apiVersion: eksctl.io/v1alpha5
+    kind: ClusterConfig
+    metadata:
+      name: my-cluster
+      region: us-east-1
+      version: "1.23"
+     
+    iam:
+      withOIDC: true
+
+    addons:
+    - name: aws-ebs-csi-driver
+      version: v1.11.4-eksbuild.1
+      attachPolicyARNs:
+      - arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy
+     
+    managedNodeGroups:
+      - name: managed-ng-2
+        instanceType: t3.medium
+        minSize: 2
+        maxSize: 3
+    ```
+
+  </Details>
```

Comment:
I added this because I found it helpful when spinning up a quick demo cluster via `eksctl`, and didn't want users in a similar boat as me to spend as much time as I did going through the AWS docs. I get what you're saying, though. Maybe we can add a red `Notice` box at the top of this `Details` box indicating that:
- This is for demo clusters only
- You should already be experienced with `eksctl`